### PR TITLE
Add notes about release issues to Plugin Release Process

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Follow up on https://github.com/mattermost/mattermost-handbook/pull/479

### CODE REVIEW

File: operations/research-and-development/engineering/plugin-release-process.md [hunk 1]
```diff
@@ -37,6 +37,14 @@ The Future Enhancements section of this document describes additional suggestion
   * Documentation through bundled releases aren’t viewable through the app and users will be looking at the latest master commit in the GitHub repo.
 * The PR for the version bump (in the plugin repo) does not mean that is the last commit to get tagged. This step only bumps the version. The tagging step actually determines the commit that is tagged with the release tag.
 
+### Create release issue
+
+* Create an issue on the corresponding repository to track the whole release process.
+  * **Title:** `Release vX.Y.Z`
+  * **Summary:** Add every outstand PR or issue as an item on as [taks list](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-task-lists).
```
Comment (justinegeffen) [inline operations/research-and-development/engineering/plugin-release-process.md]:
```suggestion
  * **Summary:** Add every outstanding PR or issue as an item on as [task list](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-task-lists).
```

**Review by justinegeffen — DISMISSED** [review]

**Comment by jfrerich** [comment]
Oh great.  The plugin release process has a landing page in the handbook.  I'll do another PR to help clean the page up and then I can stop referencing my gitbook :)

File: operations/research-and-development/engineering/plugin-release-process.md [hunk 2]
```diff
@@ -37,6 +37,14 @@ The Future Enhancements section of this document describes additional suggestion
   * Documentation through bundled releases aren’t viewable through the app and users will be looking at the latest master commit in the GitHub repo.
 * The PR for the version bump (in the plugin repo) does not mean that is the last commit to get tagged. This step only bumps the version. The tagging step actually determines the commit that is tagged with the release tag.
 
+### Create release issue
+
+* Create an issue on the corresponding repository to track the whole release process.
+  * **Title:** `Release vX.Y.Z`
+  * **Summary:** Add every outstanding PR or issue as an item on as [task list](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-task-lists).
```
Comment (jfrerich) [inline operations/research-and-development/engineering/plugin-release-process.md]:
```suggestion
  * **Summary:** Add every outstanding PR or issue as a [task list item](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-task-lists).
```

File: operations/research-and-development/engineering/plugin-release-process.md [hunk 3]
```diff
@@ -37,6 +37,14 @@ The Future Enhancements section of this document describes additional suggestion
...
+* Add that issue to the [Plugin Release board](https://github.com/orgs/mattermost/projects/2) in `Code Changes Needed`.
```
Comment (jfrerich) [inline operations/research-and-development/engineering/plugin-release-process.md]:
```suggestion
* Add that issue to the [Plugin Releases board](https://github.com/orgs/mattermost/projects/2) in `Code Changes Needed`.
```
For consistency with the actual board name.

**Review by jfrerich — DISMISSED** [review]

**Review by justinegeffen — DISMISSED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**